project-base/tests/ShopBundle/Test/DatabaseTestCase.php

```diff
@@ -5,7 +5,7 @@
 abstract class DatabaseTestCase extends FunctionalTestCase
 {
     /**
-     * @return \Doctrine\ORM\EntityManagerInterface
+     * @return \Shopsys\FrameworkBundle\Component\EntityExtension\EntityManagerDecorator
```

Comment:
We use the interface typehint throughout the application, e.g. in `AdministratorFacade`, even though in `changePassword()` method is  `$this->em->flush()` called an argument. I do agree that it should be changed to the specific implementation but you changed it only in this one place, why?